knexfile.ts

```diff
@@ -15,16 +15,19 @@ const configs: IKnexConfig = {
     client: "pg",
     connection: DB,
     migrations: {
-      directory: "./migrations",
+      directory: "./src/database/migrations",
```

Comment:
just curious why the change here

Reply:
Moved the knexfile to root, it just wouldn't put the seed folder in the right place otherwise so I had to adjust the paths